README.md

```diff
@@ -150,10 +150,13 @@ and `X-SSL-Subject` HTTP headers to have been set on the front end server.
 
 The listening port can be configured by setting `server_http_port` (which defaults to 8139).
 
-By default, this HTTP instance accepts no connection (`deny all` in the `<Directory>`
+For passenger setups, this HTTP instance accepts no connections by default (`deny all` in the `<Directory>`
 snippet). Allowed hosts can be configured by setting the `server_http_allow` parameter
 (which expects an array).
 
+For puppetserver, this HTTP instance accepts **ALL** connections and no further restrictions can be configured. The
+`server_http_allow` parameter has no effect at all!
```

Comment:
Could you add a `fail()` perhaps if server_http_allow is set to anything other than the default (`[]`) when in server_http mode with Puppet Server? It might help in case the documentation isn't seen by a user - just to be _really_ clear that it shouldn't be relied upon for security.